src/gravel/controllers/bootstrap.py

```diff
@@ -50,11 +55,16 @@ async def _is_bootstrapping(self) -> bool:
     async def bootstrap(self) -> bool:
         logger.debug("bootstrap > do bootstrap")
 
-        if not await self._should_bootstrap():
```

Comment:
Is `_should_bootstrap` redundant now and needs removing?

Reply:
yeah, missed that. thanks.